addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,523 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, models
+import logging
+from odoo import api, models, Command
```

Comment:
While we're at it, could you reorder your import so we follow the guidelines? (https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html?highlight=wizard#imports)